packages/@aws-cdk/aws-ecs-patterns/test/fargate/integ.asset-image.expected.json

```diff
@@ -776,7 +776,10 @@
               }
             ]
           }
-        }
+        },
+        "EnableECSManagedTags": true,
+        "PropagateTags": "SERVICE",
+        "ServiceRegistries": []
```

Comment:
Do we need this property to be specified even if it's empty? Can we stop it from rendering if there's no values, and what is the behaviour with CFN if it doesn't exist in the template?

Reply:
I feel like this property is not added by me. I just did the refactoring work for this. But I am trying to remove this to see if the test passes

Reply:
If it's not caused by your PR, you shouldn't have it here.